app/check.php

```diff
@@ -67,7 +67,7 @@ function_exists('xcache_set')
  */
 function check($boolean, $message, $help = '', $fatal = false)
 {
-    echo $boolean ? "  OK        " : sprintf("\n\n[[%s]] ", $fatal ? ' ERROR ' : 'WARNING');
+    echo $boolean ? '  OK        ' : sprintf("\n\n[[%s]] ", $fatal ? ' ERROR ' : 'WARNING');
     echo sprintf("$message%s\n", $boolean ? '' : ': FAILED');
 
     if (!$boolean) {
```

Comment:
This file is updated automatically from SensioDistributionBundle (by a composer hook), so please don't edit it by hand (otherwise, running ``composer install`` will leave the project in a modified state)